src/wp-includes/html-api/class-wp-html-processor-state.php

```diff
@@ -393,9 +393,6 @@ class WP_HTML_Processor_State {
 	 *  - When not in `QUIRKS_MODE`, a TABLE start tag implicitly closes an open P tag
 	 *    if one is in scope and open, otherwise the TABLE becomes a child of the P.
 	 *
-	 * `QUIRKS_MODE` impacts many styling-related aspects of an HTML document, but
-	 * none of the other changes modifies how the HTML is parsed or selected.
```

Comment:
The lines immediately above this about P > TABLE handling in quirks/no-quirks modes seem contradictory. That's directly related to tree-construction.

Reply:
I've updated the wording. It indeed was self-contradictory